dlk/python/dlk/templates/src/func/conv2d.cpp

```diff
@@ -172,14 +167,13 @@ void convolution(
 {
   // use special implementation for 1x1 conv
   if (p.kernel_height == 1 && p.kernel_width == 1 && p.padding == 0) {
-    int kernels_size = p.kernel_height * p.kernel_width * p.kernel_depth * p.output_channels;
     conv1x1_kn2row(input, kernels, output, p);
     return;
   } else if (p.kernel_height == 3 && p.kernel_width == 3 && p.padding == 1) {
     int kernels_size = p.kernel_height * p.kernel_width * p.kernel_depth * p.output_channels;
-    T* kernels_hwoi = new T[kernels_size]();
+    T* kernels_hwoi = new T[kernels_size];
     ohwi_to_hwoi(kernels, kernels_hwoi, p);
-    conv3x3_kn2row(input, kernels_hwoi, output, p);
+    conv3x3_kn2row(input, kernels_hwoi, output, p);      
```

Comment:
Please remove this kind of white space changes.

Reply:
fixed